### PR TITLE
Init the installer script

### LINKED ISSUE #65: Create an installer script
Some projects like [`box-project/box2`](https://github.com/box-project/box2) suggests to use an installer in order to automatically download the last version of the tool, set proper rights and move it to the binaries directory.

``` sh
# README.md
$ curl -L https://github.com/manala/manalize/releases/download/v0.2.1/manalize.phar > /usr/local/bin/manalize
$ chmod +x /usr/local/bin/manalize
```

On our side, it'll allow to forget about editing `README.md` at each new version, as we can do api calls on github in order to retrieve the last version automatically.

From the user's perspective, you'll only have to ~~type~~ copy/paste the following line:

``` sh
$ curl -LSs https://github.com/manala/manalize/blob/master/installer.php | php
```

We can also host the script on a dedicated repository and make it available from a more user-friendly uri, for instance: http://www.manala.io/installer.php

### PR DESCRIPTION
**Pull request by ogizanagi**
``` sh
# curl -LSs https://raw.githubusercontent.com/manala/manalize/master/installer.php | php
curl -LSs https://raw.githubusercontent.com/manala/manalize/installer/installer.php | php
```

capture d ecran 2016-10-27 a 14 23 50
capture d ecran 2016-10-27 a 14 24 11
capture d ecran 2016-10-27 a 14 24 19

(fixes #65 )

### CODE REVIEW

**Comment by chalasr** [comment]
Is this testable? :)

Really nice work! 👍

File: installer.php [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                '10.0.10586' === PHP_WINDOWS_VERSION_MAJOR.'.'.PHP_WINDOWS_VERSION_MINOR.'.'.PHP_WINDOWS_VERSION_BUILD
+                || false !== getenv('ANSICON')
+                || 'ON' === getenv('ConEmuANSI')
+                || 'xterm' === getenv('TERM');
+        }
+
+        return function_exists('posix_isatty') && @posix_isatty(STDOUT);
+    }
+}
+
```
Comment (chalasr) [inline installer.php:195]: I think we should check the PHP version here

**Comment by ogizanagi** [comment]
> Is this testable? :)

I don't know. Maybe by testing the `Installer` class by adding `installer.php` to the classmap in tests, or some kind of integration test by executing `curl -LSs https://raw.githubusercontent.com/manala/manalize/installer/installer.php | php` in a process, but I should find a way to change the target path for the executable in order to put it in a tmp dir.

File: installer.php (hunk 1 above)
Comment (ogizanagi) [reply installer.php:195]: Finally, I won't. We talked about showing how to use manalize through docker for those who don't want to install another php version locally. But they still need to install manalize on their host, so we can't restrict the script.

Comment (chalasr) [reply installer.php:195]: Hmm.. now I have some stuff in mind I would like to discuss, let's see that at office.

**Comment by ogizanagi** [comment]
@chalasr : Finally got some time to make this little test !

This is ready.

**Comment by chalasr** [comment]
Great, thank you @ogizanagi

**Pull request merged**